README.md

```diff
@@ -1,103 +1,177 @@
 
-## Creating a CloudPlatform namespace
+### Requirements
 
-When deploying to a new namespace, you may wish to use this template typescript project namespace as the basis for your new namespace:
+This application is built for node 16 and docker will be needed to run it locally.
+[nvm](https://github.com/nvm-sh/nvm) or [fnm](https://github.com/Schniz/fnm)
+can be used to install appropriate node versions.
 
-<https://github.com/ministryofjustice/cloud-platform-environments/tree/main/namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-template-typescript>
+Additional tools are required to manage deployment: kubectl and helm.
 
-This template namespace includes an AWS elasticache setup - which is required by this template project.
+### Using services in `dev` environment
 
-Copy this folder, update all the existing namespace references, and submit a PR to the CloudPlatform team. Further instructions from the CloudPlatform team can be found here: <https://user-guide.cloud-platform.service.justice.gov.uk/#cloud-platform-user-guide>
+This is probably the easiest way to run and develop on your machine: by hooking into services that already exist
+in the `dev` environment.
+A user account is needed in hmpps-auth with the appropriate roles.
 
-## Renaming from HMPPS Template Typescript - github Actions
+Copy the `.env.sample` file to `.env` following the instructions in the file.
 
-Once the new repository is deployed. Navigate to the repository in github, and select the `Actions` tab.
-Click the link to `Enable Actions on this repository`.
+Run the application in development mode, in separate shell sessions:
 
-Find the Action workflow named: `rename-project-create-pr` and click `Run workflow`.  This workflow will
-execute the `rename-project.bash` and create Pull Request for you to review.  Review the PR and merge.
+```shell
+docker compose -f docker-compose-dev.yml up
+npm run start:dev
+```
 
-Note: ideally this workflow would run automatically however due to a recent change github Actions are not
-enabled by default on newly created repos. There is no way to enable Actions other then to click the button in the UI.
-If this situation changes we will update this project so that the workflow is triggered during the bootstrap project.
-Further reading: <https://github.community/t/workflow-isnt-enabled-in-repos-generated-from-template/136421>
+This will automatically restart it if server code or front-end assets are modified.
 
-## Manually branding from template app
-Run the `rename-project.bash` and create a PR.
+### Using only local services
 
-The rename-project.bash script takes a single argument - the name of the project and calculates from it the project description
-It then performs a search and replace and directory renames so the project is ready to be used.
+**TODO:** the environment/settings are not properly set up for this application to work without using external services!
+This section should not be used at present.
```

Comment:
is it better to leave this to-do or remove the section? 🤔

Reply:
I say remove till it applies but don't mind too much